rotkehlchen/inquirer.py

```diff
@@ -344,10 +351,25 @@ def inject_evm_managers(evm_managers: Sequence[tuple[ChainID, 'EvmManager']]) ->
         for chain_id, evm_manager in evm_managers:
             instance._evm_managers[chain_id] = evm_manager
 
+    @overload
     @staticmethod
     def get_evm_manager(chain_id: ChainID) -> 'EvmManager':
+        ...
+
+    @overload
+    @staticmethod
+    def get_evm_manager(
+            chain_id: ChainID,
+            for_curve: bool,
+    ) -> 'ArbitrumOneManager | BaseManager | EthereumManager | GnosisManager | OptimismManager | PolygonPOSManager':  # noqa: E501
+        ...
+
+    @staticmethod
+    def get_evm_manager(chain_id: ChainID, for_curve: bool = False) -> 'EvmManager':
```

Comment:
What are you doing here lol. Come on. Adding an argument for type cheking? This is not how typing works. You would need to add different argument values for chain id and proper overloading. Please see how it's done in the codebase